ui/common/StatusBar.tsx

```diff
@@ -62,7 +62,7 @@ export default function StatusBar({
       className={clsx('border-t border-white/25 max-md:px-4 max-md:py-8', {
         'w-screen': full,
         'w-full': !full,
-        'bg-green-400/25': getStatus() === Status.Success,
+        'bg-[#28b12326]': getStatus() === Status.Success,
```

Comment:
Are we planning to use this color for other things apart from this element? If so, I think its better to move this to our `tailwind.config.js`. If it's a one off, ignore my comment.

Reply:
I decided to narrow the focus of this PR to just handle the success status bar color, We can continue to consider the custom tailwind config but for now lets just make sure we see green